File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, t):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        m={}
        for i,n in enumerate (nums):
            c=t-n
            if c in m:
                return [m[c],i]
            m[n]=i

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] arr=new int[2];
        HashMap <Integer,Integer> map=new HashMap<>();
        for(int i=0;i<nums.length;i++)
        {
            Integer val=map.get(target-nums[i]);
            if(val==null)
            {
                map.put(nums[i],i);
            }
            else{
                arr[0]=i;
                arr[1]=val;
            }

        }
        return arr;
    }

}   